itk_pretix.services.yml

```diff
@@ -15,3 +15,31 @@ services:
     class: Drupal\itk_pretix\Pretix\OrderHelper
     arguments:
       - '@database'
+
+  itk_pretix.exporter_manager:
+    class: Drupal\itk_pretix\Exporter\Manager
+    tags:
+      - { name: service_collector, tag: itk_pretix_exporter, call: addEventExporter }
```

Comment:
All services defined here could be defined as lazy given they only seem to be used bu logged in editors?